docs/UserGuide.md

```diff
@@ -47,6 +47,25 @@ as `nutrigoals.jar`. Do not delete or edit these files.
 
 --------------------------------------------------------------------------------------------------------------------
 
+### Screen layout
```

Comment:
I think the heading is wrong here?

<img width="1582" alt="Screenshot 2022-10-30 at 11 25 31 PM" src="https://user-images.githubusercontent.com/97275050/198886895-0995781a-8f43-4db9-8bef-3a2bfc8f6ed7.png">

Reply:
Fixed!